src/puppetlabs/dujour/version_check.clj

```diff
@@ -7,7 +7,8 @@
             [trptcolin.versioneer.core :as version]
             [slingshot.slingshot :as sling]
             [puppetlabs.kitchensink.core :as ks]
-            [clojure.set :as set]))
+            [clojure.set :as set]
+            [clojure.string :as str]))
```

Comment:
I don't think this import is required since the `str` function already exists in clojure.core.